<launch>
  <node name="dig_control_server" pkg="dig_control" type="dig_control_server" output="screen">
    <param name="debug"                        type="bool"   value="true" />
    <param name="floor_test"                   type="bool"   value="true" />
    <param name="full_autonomy"                type="bool"   value="false" />
    <param name="voltage_compensation"         type="bool"   value="false" />
    <param name="min_voltage"                  type="double" value="30.0"  />
    <param name="full_voltage"                 type="double" value="42.0"  />
    <param name="max_compensated_duty"         type="double" value="0.50"  />
    <param name="battery_filter_k"             type="double" value="0.03"  />
    <param name="central_drive_angle_filter_k" type="double" value="0.10" />
    <param name="current_filter_k"             type="double" value="0.04" />
    <param name="bucket_filter_k"              type="double" value="0.03" />
  </node>
</launch>
